README.md

```diff
@@ -21,7 +21,7 @@ Registry](https://docs.aws.amazon.com/glue/latest/dg/schema-registry-gs.html) in
 
 1. Messages/records are serialized on producer front and deserialized on the consumer front by using 
 schema-registry-serde.
-1. Support for AVRO and JSON Data formats (with [JSON Schema](https://json-schema.org/) Draft04, Draft06, Draft07).
+1. Support for three data formats: AVRO, JSON (with [JSON Schema](https://json-schema.org/) Draft04, Draft06, Draft07), and Protocol Buffers as known as Protobuf (with versions proto2 and proto3 without support for [extensions](https://developers.google.com/protocol-buffers/docs/proto#extensions) or [groups](https://developers.google.com/protocol-buffers/docs/proto#groups)).
```

Comment:
No need of mentioning syntax types and unsupported features. We can just call it Protocol Buffers (Protobuf)